### DISCUSSION TITLE
Using @map in schema re-orders field namesalphabetically?

### DISCUSSION THREAD

**Discussion by helmist** [prisma/prisma#16661] [Q&A] [answered]
when specifying @map for field names, Prisma orders mapped fields alphabetically when migrating, but does not if no map is specified. How can I stop the alphabetical re-ordering?

**Comment by nurul3101** [comment, marked as answer]
Hey @helmist 👋 

At the moment, it's not possible to control the re-ordering behaviour. Here are a few related GitHub Issues: 

- https://github.com/prisma/prisma/issues/2790
- https://github.com/prisma/prisma/issues/8283

I would recommend you to have a look at it and add a 👍 to them so that our product team can prioritise it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
